### DISCUSSION TITLE
一个修改浏览器默认字体的油猴脚本

### DISCUSSION THREAD

**Discussion by giscus** [loveminimal/comment#31] [General]
# 一个修改浏览器默认字体的油猴脚本

一个修改浏览器默认字体的油猴脚本 - https://aituyaa.com/%E4%B8%80%E4%B8%AA%E4%BF%AE%E6%94%B9%E6%B5%8F%E8%A7%88%E5%99%A8%E9%BB%98%E8%AE%A4%E5%AD%97%E4%BD%93%E7%9A%84%E6%B2%B9%E7%8C%B4%E8%84%9A%E6%9C%AC/

https://aituyaa.com/%E4%B8%80%E4%B8%AA%E4%BF%AE%E6%94%B9%E6%B5%8F%E8%A7%88%E5%99%A8%E9%BB%98%E8%AE%A4%E5%AD%97%E4%BD%93%E7%9A%84%E6%B2%B9%E7%8C%B4%E8%84%9A%E6%9C%AC/

**Comment by Ryutaku** [comment]
这个脚本用来修改苹果官方网站的字体还是很好的。官方的字体在低分辨率显示器上的显示效果实在是太差了。改为雅黑light，瞬间高大上了

**Reply by loveminimal** [reply]
最初是因为看文档的时候默认字体太丑了 🤣
